osmtm/templates/user.mako

```diff
@@ -29,6 +29,30 @@ else:
           <img src="http://www.openstreetmap.org/favicon.ico" alt="[OSM]" />${_('OSM Profile')}</a>
       </p>
     </div>
+    <div class="col-md-6">
+    % if creation_date != 'null':
+    <p title=${creation_date}>
+       ${_('This user joined OSM on ${join_date}.', mapping={'join_date':creation_date[5:7] + \
+                                                               '/' + creation_date[8:10] + '/' + \
+                                                               creation_date[0:4]})}
+    </p>
```

Comment:
Haven't forgotten about this :) got busy with work, but will get back to this soon.